qqmlsortfilterproxymodel_plugin.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <QQmlExtensionPlugin>
+
+class QQmlsortfilterproxymodelPlugin : public QQmlExtensionPlugin
```

Comment:
I think this should work though. Not sure but this file was autogenerated by QtCreator when I selected New Project >  Library > Qt Quick 2 Extension Plugin.